.github/workflows/sycl-nightly.yml

```diff
@@ -23,6 +23,19 @@ jobs:
       # prefer widespread gzip compression.
       artifact_archive_name: sycl_linux.tar.gz
 
+  ubuntu2204_shared_build:
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl-linux-build.yml
+    secrets: inherit
+    with:
+      build_cache_root: "/__w/"
+      build_cache_suffix: sprod_shared
+      build_artifact_suffix: sprod_shared
+      build_configure_extra_args: '--shared-libs --hip --cuda'
```

Comment:
```suggestion
      build_configure_extra_args: '--shared-libs --hip --cuda --native_cpu'
```

Reply:
Done in latest commit, passing in nightly [here](https://github.com/intel/llvm/actions/runs/10421733304/job/28865246621), besides unrelated proxy issue.